gen/nvml/device.go

```diff
@@ -1256,6 +1256,24 @@ func (Device Device) ResetGpuLockedClocks() Return {
 	return DeviceResetGpuLockedClocks(Device)
 }
 
+// nvmlDeviceSetMemoryLockedClocks()
+func DeviceSetMemoryLockedClocks(Device Device, MinMemClockMHz uint32, MaxMemClockMHz uint32) Return {
```

Comment:
I have probably asked this before, but are we explicitly using `uint32` here for `unsigned int` in C?

Reply:
Only when it makes sense for the type. See the discussion here:
https://github.com/NVIDIA/go-nvml/issues/31